src/test/java/teammates/test/cases/logic/FeedbackQuestionsLogicTest.java

```diff
@@ -68,12 +68,12 @@ private void testGetRecipientsForQuestion() throws Exception {
         recipients = fqLogic.getRecipientsForQuestion(question, email);
         assertEquals(recipients.size(), 2); // 3 - giver = 2
 
-        ______TS("empty case: response to team members, but alone");
+        ______TS("response to team members, total 2");
```

Comment:
Technically we are missing out on a boundary value test case. Any way to avoid this change?

Reply:
I have changed student 2 from another section to this section, causing this change. I found that there is a course 3 with only 1 student enrolled, so I can use student in course 3 to test the boundary case!

Reply:
> so I can use student in course 3 to test the boundary case!

Yes. Add back the test case